sound/soc/intel/common/soc-acpi-intel-icl-match.c

```diff
@@ -29,6 +28,13 @@ struct snd_soc_acpi_mach snd_soc_acpi_intel_icl_machines[] = {
 		.sof_fw_filename = "sof-icl.ri",
 		.sof_tplg_filename = "sof-icl-rt5682.tplg",
 	},
+	{
+		.id = "DLGS7219",
+		.drv_name = "jsl_da7219_max98373",
+		.machine_quirk = snd_soc_acpi_codec_list,
+		.sof_fw_filename = "sof-icl.ri",
+		.sof_tplg_filename = "sof-jsl-da7219.tplg",
```

Comment:
sof-icl, and create the relevant file from the same template.

Reply:
Ack.